gravitee-apim-console-webui/src/management/api/notification-settings/notofication-settings-details/api-notification-settings-details.module.ts

```diff
@@ -22,28 +22,33 @@ import { MatInputModule } from '@angular/material/input';
 import { MatCardModule } from '@angular/material/card';
 import { MatSnackBarModule } from '@angular/material/snack-bar';
 
-import { NotificationDetailsComponent } from './notification-details.component';
+import { ApiNotificationSettingsDetailsComponent } from './api-notification-settings-details.component';
 
-import { GioPermissionModule } from '../../../../../shared/components/gio-permission/gio-permission.module';
-import { GioGoBackButtonModule } from '../../../../../shared/components/gio-go-back-button/gio-go-back-button.module';
+import { GioPermissionModule } from '../../../../shared/components/gio-permission/gio-permission.module';
+import { GioGoBackButtonModule } from '../../../../shared/components/gio-go-back-button/gio-go-back-button.module';
+import { NotificationSettingsListModule } from '../../../../components/notifications/notifications-list/notification-settings-list.module';
 
 @NgModule({
-  declarations: [NotificationDetailsComponent],
-  exports: [NotificationDetailsComponent],
+  declarations: [ApiNotificationSettingsDetailsComponent],
+  exports: [ApiNotificationSettingsDetailsComponent],
   imports: [
     CommonModule,
+    ReactiveFormsModule,
+
     GioPermissionModule,
     GioIconsModule,
     GioPermissionModule,
     GioSaveBarModule,
     GioFormHeadersModule,
     GioFormFocusInvalidModule,
     GioGoBackButtonModule,
-    ReactiveFormsModule,
+
     MatCheckboxModule,
     MatInputModule,
     MatCardModule,
     MatSnackBarModule,
```

Comment:
Idem, all these imports are now not necessary because they are done by the child component module.

Reply:
Done